helm/csi-vxflexos/values.yaml

```diff
@@ -38,6 +38,11 @@ kubeletConfigDir: /var/lib/kubelet
 # Default value: none
 defaultFsType: ext4
 
+# externalAccess: allows to specify additional entries for hostAccess of NFS volumes. Both single IP address and subnet are valid entries.
```

Comment:
This line can be rephrased as: allows to specify additional entries for host to access NFS volumes.